elpis/engines/hft/objects/transcription.py

```diff
@@ -8,19 +8,19 @@
 import pympi
 import torch
 from loguru import logger
-from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor
+from transformers import Wav2Vec2ForCTC, Wav2Vec2Processor, pipeline
 from werkzeug.datastructures import FileStorage
 
 import elpis.engines.common.utilities.resampling as resampler
 from elpis.engines.common.objects.transcription import Transcription as BaseTranscription
 from elpis.engines.hft.objects.model import HFTModel
 
+LOAD_MODEL = "load_model"
 LOAD_AUDIO = "load_audio"
-PROCESS_INPUT = "process_input"
 TRANSCRIPTION = "transcription"
 SAVING = "saving"
 
-STAGES = [LOAD_AUDIO, PROCESS_INPUT, TRANSCRIPTION, SAVING]
+STAGES = [LOAD_MODEL, LOAD_AUDIO, TRANSCRIPTION, SAVING]
```

Comment:
This should be an enum.

Reply:
changed this and TRANSCRIPTION_STATUS to enums in next commit